docker-assets/manageiq.conf

```diff
@@ -1,9 +1,6 @@
 # Timeout: The number of seconds before receives and sends time out.
 Timeout 120
 
-# HTTP Start-up error log
-ErrorLog /var/log/httpd/miq_apache.log
```

Comment:
Never mind, I was thinking this was the httpd running on our appliance. This is Fine.